lib/shopify_api/resources/draft_order.rb

```diff
@@ -6,5 +6,9 @@ def send_invoice(draft_order_invoice = ShopifyAPI::DraftOrderInvoice.new)
       resource = post(:send_invoice, {}, draft_order_invoice.encode)
       ShopifyAPI::DraftOrderInvoice.new(ShopifyAPI::DraftOrder.format.decode(resource.body))
     end
+
+    def complete(options = {})
+      load_attributes_from_response(patch(:complete, {}, options.to_json))
```

Comment:
Oh so `put` in the route accepts both `put` and `patch`, but an explicit `patch` only accepts `patch` and not `put`? That's kinda an important detail. 

But yea, changing the endpoint itself sounds good to me.